external/llvm-project/mlir/lib/Dialect/Index/IR/IndexOps.cpp

```diff
@@ -652,11 +652,7 @@ static bool compareSameArgs(IndexCmpPredicate pred) {
   case IndexCmpPredicate::UGE:
   case IndexCmpPredicate::ULE:
     return true;
-  case IndexCmpPredicate::NE:
-  case IndexCmpPredicate::SGT:
-  case IndexCmpPredicate::SLT:
-  case IndexCmpPredicate::UGT:
-  case IndexCmpPredicate::ULT:
+  default:
```

Comment:
same question

Reply:
The compiler warns that some values are not covert in the switch statement.

Reply:
1. Those need to be upstream patches
2. The practice is to stick in a `default` with a `llvm_unreachable` in it